INSTALL.md

```diff
@@ -382,6 +382,15 @@ sudo zypper install --allow-downgrade -y cmake gcc gcc-c++ git gzip \
         libcurl-devel libopenssl-devel make tar wget
 ```
 
+The following steps will install libraries and tools in `/usr/local`. CentOS
+does not search for shared libraries in these directories by default, there
+are multiple ways to solve this problem, the following steps are one solution:
+
+```bash
+(echo "/usr/local/lib" ; "/usr/local/lib64") | \
```

Comment:
Does this command work? It looks like it's trying to run `"/usr/local/lib64"` as a command. Will that work?